appmanager/franca/appmgr_core.fidl

```diff
@@ -33,42 +32,33 @@ interface AppManager_Core
 			AppInfo info
 		}
 	}
-	
-	
-	
-	<** 
-		@description : this gives the current active application (the one which is currently being displayed)
-	**>
-		 
-	attribute String CurrentActiveApp readonly
-	
+
 	<**
 		@description : launch an app using the app_id. App manager resolves the app_id to an entry point and launches it.
 	**>
-	
 	method LaunchApp
 	{
 		in
 		{
 			String app_id
 		}
-		
 	}
 	
-	
 	<**
 		@description : This should signal the addition or deletion of new apps, change in capability of apps to handle mimes etc. It si adviced to call GetAppMimeTable and GetInstalledApps as needed when this broadcast is received.  
 	**>
-	
 	broadcast AppsInfoUpdated {}	
-	
-	
+
 	<**
 		@description : AppInfo is a map of manifest data. It stores Key and value as strings .The manifest should be a part of app bundle and app manager extracts and stores this data
 	**>
-	map AppInfo
+	struct AppInfo
 	{
-		String to String
+                String app_id
```

Comment:
I am not convinced that app_id should be a part of AppInfo struct. Atleast for the usecase that we have considered here, systemUI requests for app_info by providing an app_id. Why embed it again in response? For sure the app manager internally holds a structure like this so that he can fetch app_info a particular app_id but I don't see a need for it in the core interface. We should reflect on a usecase meant for this